### PR TITLE
Add a style test to enforce PEP8 compliance.

### PR DESCRIPTION
**Pull request by jeremycline**
This adds a test to enforce PEP8 compliance and fixes all the violations
in the anitya package. This does not fix the unit tests and style is not
currently enforced there. They can be incrementally fixed and added to
the enforced paths list.

Signed-off-by: Jeremy Cline <jeremy@jcline.org>

### CODE REVIEW

File: anitya/lib/__init__.py [hunk 1]
```diff
@@ -9,9 +9,7 @@
 anitya internal library.
 """
 
-
-__requires__ = ['SQLAlchemy >= 0.7']
-import pkg_resources
```
Comment (pypingou) [inline anitya/lib/__init__.py]: I believe the import is needed for the ``__requires__`` to work, no?

File: anitya/ui.py [hunk 2]
```diff
@@ -252,7 +252,7 @@ def projects_search(pattern=None):
             SESSION, pattern=get_extended_pattern(pattern), count=True)
     else:
         projects_count = anitya.lib.model.Project.search(
-            SESSION, pattern=pattern, distro=distroname, count=True)
+            SESSION, pattern=pattern, count=True)
```
Comment (pypingou) [inline anitya/ui.py]: This is an odd left over

**Comment by pypingou** [comment]
2 questions otherwise looks good

**Comment by jeremycline** [comment]
Good catch on both of those, I've never used pkg_resources before and didn't realize importing it had a side-effect.

**Comment by pypingou** [comment]
:+1: for me

**Pull request merged**